gnome/gingerblue/Portfile

```diff
@@ -23,9 +23,9 @@ master_sites        gnome:sources/${name}/${branch}/
 
 use_xz              yes
 
-checksums           rmd160  d1482a8ac4b1fd863f1f4edc267cc7e87e00a49f \
-                    sha256  118dc659d229e2fa7e6c03c8b7aef738f0e9e7e773cc941329b5afbeb82e88a9 \
-                    size    320484
+checksums           rmd160  56a9c1b51324ad82395e683da17ff7164117b0f1 \
+                    sha256  dd63cd1442fd9b8c0c6b31a9c9dc68d9a992a72c0642b0d7942889d8eb280f27 \
+                    size    321940 
 
 depends_build       port:autoconf \
                     port:automake \
```

Comment:
The autoconf and automake dependencies should be removed. The port doesn't use them; it uses the pre-generated build system shipped in the tarball. (If the port were regenerating the build system with auto(re)conf, then the dependencies would be needed, and the intltool patch would no longer be needed since the intltool port includes a fix for the intltool bug.)

As far as I can tell, the itstool dependency isn't used either and should be removed too. itstool is mentioned in the gingerblue.spec.in and gingerblue.spec files but nowhere else that I can see.

Dependencies that are mentioned in depends_lib should be removed from depends_build since that's redundant.